lib/mpl_toolkits/mplot3d/proj3d.py

```diff
@@ -6,31 +6,27 @@
 import numpy.linalg as linalg
 
 
-def _line2d_seg_dist(p1, p2, p0):
+def _line2d_seg_dist(s0, s1, p):
```

Comment:
```suggestion
def _line2d_seg_dist(p, s0, s1):
```
Do you want to reorder this to be the same argument order as the `dist_point_to_segment()` since it is a private helper?

Reply:
I think the two functions are far enough from one another that consistency doesn't really matter here?

Reply:
Sure, it doesn't make a big difference, the text description started with the final argument which seemed odd. No big deal either way from me.

Reply:
Fair enough, swapped the order.